src/js/reducers/cycleCountReducer.js

```diff
@@ -34,6 +35,11 @@ export default function cycleCountReducer(state = initialState, action) {
         ...state,
         binLocations: action.payload,
       };
+    case ERASE_DRAFT:
+      return {
+        ...state,
+        requests: [],
+      };
```

Comment:
I'm totally fine with this solution, but is using an action required here? All we're doing is clearing the requests prop. Seems like chaining that through the ERASE_DRAFT action is a bit redundant. I'm far from an expert in React though, so doing it as you have it might be more correct.

Reply:
It's how Redux's flow is working, so it's not redundant even though it might look like this.